packages/e2e/src/specs/contacts/vcf-files/52Contacts-E2E-tests.vcf

```diff
@@ -0,0 +1,616 @@
+BEGIN:VCARD
```

Comment:
Just wondering - are those real data or just example found somewhere on the Internet?